lhotse/audio.py

```diff
@@ -470,13 +475,7 @@ def move_to_memory(
             channels=channels, offset=ifnone(offset, 0), duration=duration
         )
         stream = BytesIO()
-        if torchaudio_soundfile_supports_format() and format in (
-            "wav",
-            "ogg",
-            "vorbis",
-            "flac",
-            "sph",
-        ):
+        if torchaudio_soundfile_supports_format() and format == "flac":
             # Prefer saving with soundfile backend whenever possible to avoid issue:
             # https://github.com/pytorch/audio/issues/2662
             # Saving with sox_io backend to FLAC may corrupt the file, IDK about other
```

Comment:
So, other formats (except flac) can be safely saved with the sox_io backend?

Reply:
I believe so, at least I never experienced an issue with a different format.